tiledb/sm/c_api/tiledb.cc

```diff
@@ -474,15 +500,83 @@ int32_t tiledb_array_schema_load(
 
   if (uri.is_tiledb()) {
     auto& rest_client = ctx->context().rest_client();
-    auto&& [st, array_schema_rest] =
-        rest_client.get_array_schema_from_rest(uri);
-    if (!st.ok()) {
+    try {
+      auto array_schema_response = rest_client.post_array_schema_from_rest(
+          ctx->resources().config(), uri, 0, UINT64_MAX, false);
```

Comment:
Could we clarify the doxygen of `tiledb_array_schema_load` to explicitely mention we are always returning the latest array schema of the array ?